inferno-vc/src/Inferno/VersionControl/Operations/Filesystem.hs

```diff
@@ -177,7 +191,7 @@ instance (MonadIO m, MonadMask m, AsType VCStoreError err) => InfernoVCOperation
     lock <- asks getTyped
     withWrite lock $ do
       -- check if object meta exists with hash meta_hash, and get meta
-      (VCMeta {name = obj_name} :: VCMeta Value Value VCObject) <- fetchVCObject obj_hash
+      VCMeta {name = obj_name} <- fetchVCObject obj_hash
```

Comment:
This change is made possible because `deleteAutosavedVCObject` now knows what `g` and `a` are so we don't have to "upcast"